default/stringtables/ru.txt

```diff
@@ -2187,7 +2268,7 @@ OPTIONS_APPLY
 Применить
 
 OPTIONS_FLUSH_STRINGTABLE
-Очистка строковой таблицы
+Очистка файла stringtables
```

Comment:
@geoffthemedio,   can you explain what `OPTIONS_FLUSH_STRINGTABLE` do?  Did it just reload stringtable from the disc?